app/services/publish_finder.rb

```diff
@@ -10,14 +10,34 @@ def self.call(finder_content_item)
   end
 
   def call
+    send_to_publishing_api
+    send_to_rummager
+  end
+
+private
+
+  def send_to_publishing_api
     Whitehall.publishing_api_v2_client.put_content(
       content_id,
       finder_content_item
     )
     Whitehall.publishing_api_v2_client.publish(content_id, "major")
   end
 
-private
+  def send_to_rummager
+    index = Whitehall::SearchIndex.for(:government)
+    index.add(present_for_rummager)
+  end
+
+  def present_for_rummager
+    {
+      _id: finder_content_item.fetch("base_path"),
+      link: finder_content_item.fetch("base_path"),
+      format: "finder",
+      title: finder_content_item.fetch("title"),
+      description: finder_content_item.fetch("description", ""),
```

Comment:
Can you add content_store_document_type, publishing_app, rendering_app, content_id too? They're new.

And probably also `indexable_content` (unless that's part of rummager's search thing already, I forgot).